.circleci/docker/common/install_breakpad.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -ex
+
+git clone https://github.com/google/breakpad.git
```

Comment:
It would be good to pin to a specific version  here, otherwise we are bound to run into a regressions